### PR TITLE
Provide better compatibility with alternate OIDC providers (Docs PR)

### PR DESCRIPTION
**Pull request by davidfestal**
### What does this PR do?

This PR is the Documentation PR for the Che PR https://github.com/eclipse/che/pull/11090
It mainly explains how to enable to new *fixed redirect URL* mode for alternate OIDC providers that don't support redirect URLs with wildcards.

### What issues does this PR fix or reference?

This PR results from the work done on issue [#10998](https://github.com/eclipse/che/issues/10998)

### CODE REVIEW

**Comment by davidfestal** [comment]
@eivantsov @slemeur Should I wait for this PR to be approved before merging the related code PR (https://github.com/eclipse/che/pull/11090)  ?

**Review by ghost — APPROVED** [review]

**Comment by slemeur** [comment]
@davidfestal That's the other way around :) You can merge the feature on che and then you can merge this pr

**Review by slemeur — APPROVED** [review]

File: src/main/pages/setup-openshift/openshift-config.adoc
Comment (rkratky) [inline src/main/pages/setup-openshift/openshift-config.adoc]: I suggest using:

_By default, the OIDC authentication in Che requires using redirect URLs with wildcards. For alternate OIDC providers that only support a list of fixed redirect URLs, use the following environment variable to switch to fixed redirect URL mode:_

File: src/main/pages/setup-openshift/openshift-config.adoc
Comment (rkratky) [inline src/main/pages/setup-openshift/openshift-config.adoc]: _In this case, the two redirect URLs that should be registered in the OIDC provider configuration are:_

File: src/main/pages/setup-openshift/openshift-config.adoc
Comment (davidfestal) [reply src/main/pages/setup-openshift/openshift-config.adoc]: Fixed in commit https://github.com/eclipse/che-docs/pull/509/commits/0d2262ef42c188c6305016283ad5f5b347b360c7

File: src/main/pages/setup-openshift/openshift-config.adoc
Comment (davidfestal) [reply src/main/pages/setup-openshift/openshift-config.adoc]: Fixed in commit https://github.com/eclipse/che-docs/pull/509/commits/0d2262ef42c188c6305016283ad5f5b347b360c7

**Comment by davidfestal** [comment]
@rkratky is it OK for you ?
@slemeur I assume you will merge this PR when the corresponding code PR will have been integrated into the next release ?

**Comment by rkratky** [comment]
> @rkratky is it OK for you ?

Yes, thank you.

**Comment by ibuziuk** [comment]
@rkratky looks PR is neglected, how should we proceed with it ?

**Comment by slemeur** [comment]
@davidfestal is this still accurate? I think we should merge it...

**Comment by davidfestal** [comment]
I assume it is still accurate. But TBH I've not tested this for a while. However I don't see why it wouldn't.

**Comment by slemeur** [comment]
Fixed by https://github.com/eclipse/che-docs/pull/762

**Pull request closed without merging**
